jupyter_ydoc/ydoc.py

```diff
@@ -153,9 +155,9 @@ def unobserve(self) -> None:
         self._subscriptions = {}
 
 
-class YFile(YBaseDoc):
+class YUTF8(YBaseDoc):
```

Comment:
> Should we use something more generic like YText?

Maybe, although `YText` already exists in [y-py](https://ypy.readthedocs.io/en/latest/autoapi/y_py/index.html#y_py.YText), but you're right, if they don't call it `YUTF8` there then I guess there is no reason that we do it here.

> the idea is a bit like for YBytes to let the door open to other encoding. And also it is probably more easy to grasp.

Actually I'd like to rename `YBytes` to `YBlob`, which reflects the idea that it's a whole chunk of data on which we cannot collaborate on an element basis. Also, it can store anything, not only bytes.

Reply:
https://github.com/jupyter-server/jupyter_ydoc/pull/122/commits/04a7f74b12c08ed5b9018d9e3a78c876fcb74473 renames `YBytes` to `YBlob`. Actually, it really stores bytes, which is fine since blob stands for "binary large object".

Reply:
Thanks for pointing the existing `YText` in y-py. So maybe `YText` here is not the most appropriate as it can generate confusion. I'm a bit short of idea - I thought about `YStr` or `YString` but i'm half convinced.

Reply:
Traitlets has [Unicode](https://traitlets.readthedocs.io/en/stable/trait_types.html#traitlets.Unicode). It's similar to the previous `YUTF8`, but maybe more inline with the rest of the Jupyter ecosystem?